diff --git a/code/CMakeLists.txt b/code/CMakeLists.txt
index 08efd28df..e7cab6dc6 100644
--- a/code/CMakeLists.txt
+++ b/code/CMakeLists.txt
@@ -1168,7 +1168,6 @@ TARGET_USE_COMMON_OUTPUT_DIRECTORY(assimp)
 IF (MSVC)
   TARGET_COMPILE_OPTIONS(assimp PRIVATE /WX)
 ELSE()
-  TARGET_COMPILE_OPTIONS(assimp PRIVATE -Werror)
 ENDIF()
 
 # adds C_FLAGS required to compile zip.c on old GCC 4.x compiler
diff --git a/contrib/zip/CMakeLists.txt b/contrib/zip/CMakeLists.txt
index bba4e7bde..98842b5e5 100644
--- a/contrib/zip/CMakeLists.txt
+++ b/contrib/zip/CMakeLists.txt
@@ -45,7 +45,7 @@ if (MSVC)
 elseif ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU" OR
         "${CMAKE_C_COMPILER_ID}" STREQUAL "Clang" OR
         "${CMAKE_C_COMPILER_ID}" STREQUAL "AppleClang")
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -Wall -Wextra -Werror -pedantic -Wno-deprecated")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -Wall -Wextra -pedantic -Wno-deprecated")
 endif (MSVC)
 
 ####
